include/grpc/support/port_platform.h

```diff
@@ -165,6 +165,7 @@
 #define GPR_WINDOWS_STAT
 #define GPR_WINDOWS_STRING
 #define GPR_WINDOWS_TIME
+#define GRPC_HAVE_UNIX_SOCKET 1
```

Comment:
The network-related defines are mostly in `src/core/lib/iomgr/port.h`, and you have defined it in both places. I suggest only defining this macro in `port.h`, and ensuring that `port.h` is included in every file that uses it.

Reply:
Removed. port.h is used when needed.